models/workload.go

```diff
@@ -61,6 +61,11 @@ type WorkloadListItem struct {
 	// required: true
 	// example: true
 	VersionLabel bool `json:"versionLabel"`
+
+	// Number of current workload pods
+	// required: true
+	// example: 1
+	Pods int `json:"pods"`
```

Comment:
Please, don't use Pods field in the WorkloadItem struct, as these is a composition with the Workload struct where there is already defined a Pods field.
Just change it PodsSize/PodsNumber or anything to avoid a clash on the naming.
Probably this will break something in other parts of the code.